arm-sql/2015-05-01-preview/swagger/failoverGroups.json

```diff
@@ -0,0 +1,600 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2015-05-01-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/failoverGroups/{failoverGroupName}": {
+      "get": {
+        "tags": [
+          "FailoverGroups"
+        ],
+        "description": "Gets a failover group.",
+        "operationId": "FailoverGroups_Get",
+        "x-ms-examples": {
+          "Get failover group": {
+            "$ref": "../examples/FailoverGroupGet.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
```

Comment:
nit: For consistency, put ```SubscriptionId``` and ```ApiVersion``` together at the beginning or at the end of the parameters section